File: solutions/scala/raindrops/7/src/main/scala/Raindrops.scala
object Raindrops:
   def convert(n: Int): String =
      val s = Seq(3 -> "Pling", 5 -> "Plang", 7 -> "Plong").collect { case (f, s) if n % f == 0 => s }.mkString
      if s.isEmpty then n.toString else s

File: solutions/r/raindrops/2/raindrops.R
raindrops <- function(number) {
  sounds <- c(Pling = 3, Plang = 5, Plong = 7)
  res <- paste0(names(sounds[number %% sounds == 0]), collapse = "")
  if (nchar(res) == 0) as.character(number) else res
}
